src/tcgwars/logic/impl/gen3/PopSeries3.groovy

```diff
@@ -176,7 +188,12 @@ public enum PopSeries3 implements LogicCardInfo {
           energyCost C
           attackRequirement {}
           onAttack {
-
+            if (all.findAll.findAll {it.name == "Plusle"}) {
```

Comment:
Is `findAll.findAll` a mistake, or something I'm not familiar with that you have to do it this way?

Reply:
It's a mistake 🤣   Thanks for catching it